Assets/Scripts/UI/RhythmUI.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RhythmUI : MonoBehaviour
+{
+    [SerializeField] private GameObject linePrefab;
+    [SerializeField] private int numOfBeats = 5;
+
+    private float _distance;
+    private RhythmTimer _time;
+    private List<RhythmLine> _lines = new();
+
+    private void Awake()
+    {
+        _distance = GetComponent<RectTransform>().sizeDelta.x / 2;
+        _time = GameObject.Find("Game Manager").GetComponent<RhythmTimer>();
+        InitialDraw();
```

Comment:
În Awake cel mai bine e doar să initializezi chestii, nu să le și folosești.

InitialDraw() apelează CreateLine() care apelează _time.GetTime() care scoate proprietatea time de pe _track din RhythmTimer, dar _track e setat și el în Awake. Dacă Awake-ul de pe RhythmUI e apelat înainte de cel de pe RhythmTimer, tot lanțul ăsta crapă în GetTime.

De-aia InitialDraw stătea corect într-un Start.

Reply:
Legat de asta, aici avusesem acel bug cu editor-ul functionand diferit fata de build. D-aia ajunsesem sa bag astea toate in awake, nu mi-am dat seama ca unul din ele trebuia sa ramana in start